docs/source/migration.mdx

```diff
@@ -189,9 +189,9 @@ If you used the `apollo-server-express` package in Apollo Server 3, use the `exp
 
 To migrate from Apollo Server 3's `apollo-server-express` package to using the `expressMiddleware` function, do the following:
 
-1. Install the `@apollo/server`, `cors`, and `body-parser` packages.
+1. Install the `@apollo/server` and `cors` packages.
 2. Import symbols from `@apollo/server` (i.e., instead of from `apollo-server-express` and `apollo-server-core`).
-3. Add `cors` and `bodyParser.json()` to your server setup.
+3. Add `cors` to your server setup.
```

Comment:
We should probably advise they update `express` to `>=4.16.0` (this is listed in their changelog as when `express.json()` was introduced)

Reply:
Good call! https://github.com/apollographql/apollo-server/pull/7690/commits/c142fca08e98b6bdf78868a24a7aa49d6769712c